### DISCUSSION TITLE
My account has been suspended for 7 days, but I still can't log in.

### DISCUSSION THREAD

**Discussion by senula8982** [community/community#145736] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My other github account was suspended for 7 days due to some mistake of mine but it has been almost a month now and I still can't log in to it. How can i fix it? I have sent an email to github support but still no response.

**Comment by mecodeatlas** [comment]
Hi @senula8982! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
